server/main.lua

```diff
@@ -80,35 +81,41 @@ local function calculateNewFinance(paymentAmount, vehData)
 end
 
 local function generateUniquePlate()
```

Comment:
Creating a new vehicle using qbx_vehicles generated a unique plate so this function shouldn't exist, and therefore we shouldn't have a use case for modifying the qbx_vehicles repo

Reply:
since `exports.qbx_vehicles:CreatePlayerVehicle` doesn't return the created plate, but only the inserted id, should i modify `exports.qbx_vehicles:CreatePlayerVehicle` in qbx_vehicles to return generated plater or register hook and use it inside function to get the plate.
the changes i made with qbx_vehicles, it used on `transfervehicle` command to get player vehicles by plate, also used `qbx_vehicleshop:server:GetVehiclesByName` event to select the plate

Reply:
If you need to get a vehicle's plate, you can do so using https://docs.qbox.re/resources/qbx_vehicles/exports/server#getplayervehicle

However, transfer vehicle shouldn't need to use the plate for any purpose. Everything should use vehicleIds.

Reply:
> If you need to get a vehicle's plate, you can do so using https://docs.qbox.re/resources/qbx_vehicles/exports/server#getplayervehicle
> 
yeah, plate is not in filters, so we cant get by plate, but i made changes now when we don't need it, but we still need to select the plate